### PR TITLE
Expand security considerations

### PR DESCRIPTION
**Pull request by chris-wood**
This adds some additional security considerations, touching on privacy considerations at the same time. Do we want to say anything about IP fragmentation, or at least point to some existing guidance around how to avoid sharp edges there?

Please feel free to reframe or modify the text as you see fit!

### CODE REVIEW

File: draft-ietf-masque-connect-ip.md
Comment (chris-wood) [inline draft-ietf-masque-connect-ip.md:28]: Sorry... this just seemed a bit too silly!

**Review by mirjak — APPROVED** [review]

**Review by achernya — APPROVED** [review]

File: draft-ietf-masque-connect-ip.md
Comment (DavidSchinazi) [reply draft-ietf-masque-connect-ip.md]: I've put this back, it's an intentional statement about the pointlessness of keywords

**Review by DavidSchinazi — APPROVED** [review]

File: draft-ietf-masque-connect-ip.md
Comment (tfpauly) [inline draft-ietf-masque-connect-ip.md:1158]: Perhaps we should call out specific header names, since both Authorization and Proxy-Authorization could be used here.

Comment (chris-wood) [reply draft-ietf-masque-connect-ip.md:1158]: I don't think it's necessary, but I defer to y'all as the editors to do whatever you think is best!

Comment (DavidSchinazi) [reply draft-ietf-masque-connect-ip.md:1158]: I propose we keep the current text and wait for IETF LC / IESG review to bikeshed these examples further

Comment (gloinul) [reply draft-ietf-masque-connect-ip.md:1158]: I would think that proxy-authorization would not be appropriate to use, as the authorization required is related to the HTTP target, not an HTTP proxy before it (for which Prozy-Authorization would be appropriate).

**Review by gloinul — APPROVED** [review]

**Pull request merged**
